examples/05-plotting/00-basic_plotting.py

```diff
@@ -6,12 +6,19 @@
 This example lists the different plotting commands available,
 shown with the arguments available.
 
+.. note::
+    This example requires the Premium ServerContext.
```

Comment:
if basic plotting requires premium, we need to create another example which doesn't to not loose the customer here

Reply:
@cbellot000 `dpf.Operator("split_mesh")` is not available in Entry. I'll change the example to remove the need for it.